public/js/publink.js

```diff
@@ -5,12 +5,33 @@ var valid_dataset = false;
 var last_retrieved = { dataset: "", publication: "" };
 
 $(document).ready(function() {
-    $('#retrieve_publication').button().click(function () {
-        retrievePublicationCitation();
+    jQuery.validator.addMethod("doiFormat", function(value, element) {
```

Comment:
I think we should stop adding more jQuery stuff, if we are planning to move away from it. We can use vanilla JS and packages for it, for validation we can use this alternative
https://validatejs.org/

Reply:
at some point, this is going to be a big epic

Reply:
I'm all for no more jQuery, html 5 input have `pattern` property.